.github/workflows/e2e-pipeline.yml

```diff
@@ -34,19 +50,19 @@ jobs:
             ${{ secrets.KAKAROT_STAGING_PRIVATE_KEY }}
         run: |
           cd scripts
-          sh upgrade.sh deploy test --staging
+          sh e2e_pipeline.sh ${{ github.event.inputs.action }} --${{ github.event.inputs.environment }}
       - name: Check diff in deployments
         run: |
           cd lib/kakarot
           set +e
-          git diff --no-patch --exit-code -- deployments/kakarot-staging/
+          git diff --no-patch --exit-code -- deployments/
           has_diff=$?
           set -e
           echo "has_diff=$has_diff" >> $GITHUB_ENV
       - name: Create PR for deployment updates
         if: ${{ env.has_diff == 1 }}
         env:
-          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is the default GITHUB_TOKEN allowed to PR on kakaort ? I don't think so, but we know have a kakarot bot that you can use, see also the caro update workflows

Reply:
what is the status of this

Reply:
working on it now that I have some time to focus on the CI and the release